README.md

```diff
@@ -145,16 +145,36 @@ There is only a limited number of such operations and they are only implemented
 ### Time Graph Tooltip
 Currently, the **Time Graph Tooltip** is populated when selecting a state in a Time Graph view.
 
+## New contributors
+⚠️ **Linux only!** The development environment for this project only works on Linux.
```

Comment:
Please add "currently" in the sentence. Eventually it should be available on other platforms.